specs/experimental/cannon-fault-proof-vm-mt.md

```diff
@@ -40,7 +40,6 @@
 - [Syscalls](#syscalls)
   - [Supported Syscalls](#supported-syscalls)
   - [Noop Syscalls](#noop-syscalls)
-- [I/O](#io)
```

Comment:
Looks like this section got cut for some reason

Reply:
Thanks for flagging, this is because I didn't keep the empty line before the section when I added the cspell ignore tags. Fixed now!